profile/README.md

```diff
@@ -14,3 +14,149 @@ The OSTIS technology is implemented as a network of ostis-systems, which is part
+
+To create an intelligent system, we need to integrate various types of knowledge and various problem-solving models, since it's unknown which set of models and knowledge will be necessary to solve a specific class of problems.
+<p>
+<table>
+  <tr>
+    <td>Various types of knowledge</td>
+    <td>Various problem solving models</td>
+  </tr>
+  <tr>
+    <td>
+    <ul>
+      <li>facts</li>
+      <li>logical statements</li>
+      <li>situations</li>
+      <li>events</li>
+      <li>algorithms</li>
+      <li>etc.</li>
+    </ul>
+    </td>
+    <td>
+      <ul>
+         <li>logical models</li>
+         <li>statistical models</li>
+         <li>neural networks</li>
+         <li>classical algorithms and genetic algorithms</li>
+         <li>etc.</li>
+      </ul>
+    </td>
+  </tr>
+ </table>
+</p>
+
+There are systems that partially solve the problem of such an integration (hybrid intelligent systems), but currently they have several disadvantages:
+
+- Such systems are made monolithic, so they are very difficult to modify
+- It is practically impossible to reuse such systems (or components of such systems) to solve problems of a completely different class; you have to develop all the components anew.
+- The development of such systems costs enormous resources.
+
+It's not hard to notice that the main problem is expandability of such systems. Thus, we need a framework to design intelligent systems that are compatible with each other, because interaction between intelligent systems is one of the ways to enable intelligent systems to solve new classes of problems. OSTIS is a technology to create **semantically compatible** intelligent systems.
+
+## Principles
+
+### **SC-code**
+
+The OSTIS Technology is based on a universal way to represent information, named SC-code (Semantic Computer Code). The SC-code is based on the basic formalisms of discrete mathematics (set theory and graph theory).
+
+That ensures:
+
+- **Universality** and **uniformity** of information representation (any information can be represented in the same way).
+
+- The convenience of machine processing and human perception.
+
+The SC-code is a **standard** for universal sense representation of information in the memory of computer systems.
+
+SC-code features:
+
+- The SC-code is non-linear, so it can be used to encode information in the memory of semantic associative computers.
+- The basic alphabet of the SC-code consists of only 5 elements, on the basis of which more and more complex structures are built.
+- You can use the SC-code to represent not only the knowledge of the system but also the problem-solving models and system interface.
+- The SC-code is abstract language and can be visualized in different forms. Currently, there are several forms of the SC-code: SCg-, SCn-, SCs-code.
+
+![Simple logic rule in SCg](docs/scg-examples/scg-logic-rule.jpeg)
```

Comment:
can'r see the rule
![image](https://github.com/ostis-ai/.github/assets/36566977/c3923e08-b80f-4afe-b603-ccfda44c99a9)

Reply:
after merge will work, difference in repositories: ostis-ai and smousy
![image](https://github.com/ostis-ai/.github/assets/37049225/075fa409-0161-4899-8eb2-ae7aaffb7703)